### ISSUE TITLE
File missing when building with Qt.py

### ISSUE THREAD

**Issue by Rittel** [pyinstaller/pyinstaller#4218]
I am currently working on a project, that I have previously built with PyQt5, which was not a problem. Now I changed this to PySide2 and used Qt.py (https://github.com/mottosso/Qt.py) as a wrapper to be able to easily switch out the qt implementation in the future.

As I was trying to build the project I keep getting the error:
'Qt' is not a package

When checking the files in the corresponding Temp-Folder I noticed that the Qt.py file is missing from that folder.
I have tried including it via imports, as file directly and also via runtime hooks. But I always end up with the same error.

During the build the file is being processed, as I find some entries in the warn-file.txt concerning some references (eventhough these should not be a problem, I attached the file)
[warn-start_script.txt](https://github.com/pyinstaller/pyinstaller/files/3153028/warn-start_script.txt)

Here is also the spec file:
[start_script.spec](https://github.com/pyinstaller/pyinstaller/files/3153030/start_script.txt)

Any ideas on how I can make this work?

**Comment by bjones1** [comment]
Build with `--debug=all` to get a better view of what's being included and what's going wrong.

**Comment by bjones1** [comment]
This might be fixed in 5b5791285332003940f4bc70c32bb0cf10a32917. Please reopen if there are still problems.

**Comment by mikebourbeauart** [comment]
I'm also having this problem. I have PySide 2 pip installed and am using Qt.py as a wrapper that I link to in the .spec file.

Here's the log from running the exe file: 

```
Traceback (most recent call last):
  File "mb-armada\main.py", line 7, in <module>
  File "d:\gitprojects\mb-armada\venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 623, in exec_module
    exec(bytecode, module.__dict__)
  File "mb-armada\source\mb_armada\gui\main_window.py", line 11, in <module>
  File "d:\gitprojects\mb-armada\venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 623, in exec_module
    exec(bytecode, module.__dict__)
  File "mb-armada\source\mb_armada\gui\main_tab_bar.py", line 17, in <module>
  File "d:\gitprojects\mb-armada\venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 623, in exec_module
    exec(bytecode, module.__dict__)
  File "mb-armada\source\mb_utils\__init__.py", line 1, in <module>
  File "d:\gitprojects\mb-armada\venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 623, in exec_module
    exec(bytecode, module.__dict__)
  File "mb-armada\source\mb_utils\logger\__init__.py", line 1, in <module>
  File "d:\gitprojects\mb-armada\venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 623, in exec_module
    exec(bytecode, module.__dict__)
  File "mb-armada\source\mb_utils\logger\logger.py", line 8, in <module>
  File "d:\gitprojects\mb-armada\venv\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 623, in exec_module
    exec(bytecode, module.__dict__)
  File "mb-armada\source\mb_utils\resource.py", line 9, in <module>
ModuleNotFoundError: No module named 'Qt.QtSvg'; 'Qt' is not a package
[1548] Failed to execute script main
PS D:\GitProjects\mb-armada\dist\armada>
```

Here's the log from running pyinstaller:

```
(venv) D:\GitProjects\mb-armada>pyinstaller armada.spec
55 INFO: PyInstaller: 3.6
56 INFO: Python: 3.7.0
56 INFO: Platform: Windows-10-10.0.19041-SP0
58 INFO: UPX is not available.
60 INFO: Extending PYTHONPATH with paths
['D:\\GitProjects',
 'D:\\GitProjects\\mb-armada',
 'D:\\GitProjects\\mb-armada\\source',
 'D:\\GitProjects\\mb-armada\\dependencies\\Qt.py',
 'D:\\GitProjects\\mb-armada\\venv\\Lib\\site-packages',
 'D:\\GitProjects\\mb-armada\\dependencies\\watchdog\\src']
60 INFO: checking Analysis
60 INFO: Building Analysis because Analysis-00.toc is non existent
61 INFO: Initializing module dependency graph...
65 INFO: Caching module graph hooks...
69 INFO: Analyzing base_library.zip ...
2958 INFO: Caching module dependency graph...
3063 INFO: running Analysis Analysis-00.toc
3066 INFO: Adding Microsoft.Windows.Common-Controls to dependent assemblies of final executable
  required by d:\gitprojects\mb-armada\venv\scripts\python.exe
3142 INFO: Analyzing main.py
4676 INFO: Processing module hooks...
4676 INFO: Loading module hook "hook-encodings.py"...
4773 INFO: Loading module hook "hook-pydoc.py"...
4775 INFO: Loading module hook "hook-PySide2.py"...
5016 INFO: Loading module hook "hook-PySide2.QtCore.py"...
5133 INFO: Loading module hook "hook-PySide2.QtGui.py"...
5312 INFO: Loading module hook "hook-PySide2.QtNetwork.py"...
qt.network.ssl: QSslSocket: cannot resolve SSL_CTX_set_ciphersuites
qt.network.ssl: QSslSocket: cannot resolve SSL_set_psk_use_session_callback
5607 INFO: Loading module hook "hook-PySide2.QtWebEngineWidgets.py"...
6378 INFO: Loading module hook "hook-PySide2.QtWidgets.py"...
6599 INFO: Loading module hook "hook-xml.etree.cElementTree.py"...
6600 INFO: Loading module hook "hook-xml.py"...
6822 INFO: Loading module hook "hook-PySide2.QtPrintSupport.py"...
7022 INFO: Loading module hook "hook-PySide2.QtQml.py"...
7297 INFO: Loading module hook "hook-PySide2.QtQuick.py"...
7680 INFO: Loading module hook "hook-PySide2.QtQuickWidgets.py"...
8240 INFO: Looking for ctypes DLLs
8257 INFO: Analyzing run-time hooks ...
8260 INFO: Including run-time hook 'pyi_rth_pyside2.py'
8261 INFO: Including run-time hook 'pyi_rth_pyside2webengine.py'
8266 INFO: Looking for dynamic libraries
8763 WARNING: lib not found: api-ms-win-core-winrt-string-l1-1-0.dll dependency of D:\GitProjects\mb-arm
ada\venv\Lib\site-packages\PySide2\plugins\position\qtposition_winrt.dll
8765 WARNING: lib not found: api-ms-win-core-winrt-l1-1-0.dll dependency of D:\GitProjects\mb-armada\ven
v\Lib\site-packages\PySide2\plugins\position\qtposition_winrt.dll
10312 WARNING: lib not found: shiboken2.abi3.dll dependency of D:\GitProjects\mb-armada\venv\Lib\site-pa
ckages\PySide2\QtWebEngineWidgets.pyd
10330 WARNING: lib not found: shiboken2.abi3.dll dependency of D:\GitProjects\mb-armada\venv\Lib\site-pa
ckages\PySide2\QtWebChannel.pyd
10358 WARNING: lib not found: shiboken2.abi3.dll dependency of D:\GitProjects\mb-armada\venv\Lib\site-pa
ckages\PySide2\QtQml.pyd
10380 WARNING: lib not found: shiboken2.abi3.dll dependency of D:\GitProjects\mb-armada\venv\Lib\site-pa
ckages\PySide2\QtPrintSupport.pyd
10396 WARNING: lib not found: shiboken2.abi3.dll dependency of D:\GitProjects\mb-armada\venv\Lib\site-pa
ckages\PySide2\QtQuick.pyd
10419 WARNING: lib not found: shiboken2.abi3.dll dependency of D:\GitProjects\mb-armada\venv\Lib\site-pa
ckages\PySide2\QtQuickWidgets.pyd
10439 WARNING: lib not found: shiboken2.abi3.dll dependency of D:\GitProjects\mb-armada\venv\Lib\site-pa
ckages\PySide2\QtPositioning.pyd
10456 WARNING: lib not found: shiboken2.abi3.dll dependency of D:\GitProjects\mb-armada\venv\Lib\site-pa
ckages\PySide2\QtWebEngineCore.pyd
10478 WARNING: lib not found: shiboken2.abi3.dll dependency of D:\GitProjects\mb-armada\venv\Lib\site-pa
ckages\PySide2\QtWidgets.pyd
10499 WARNING: lib not found: shiboken2.abi3.dll dependency of D:\GitProjects\mb-armada\venv\Lib\site-pa
ckages\PySide2\QtGui.pyd
10523 WARNING: lib not found: shiboken2.abi3.dll dependency of D:\GitProjects\mb-armada\venv\Lib\site-pa
ckages\PySide2\QtCore.pyd
10542 WARNING: lib not found: shiboken2.abi3.dll dependency of D:\GitProjects\mb-armada\venv\Lib\site-pa
ckages\PySide2\QtNetwork.pyd
11403 WARNING: lib not found: api-ms-win-core-winrt-string-l1-1-0.dll dependency of d:\gitprojects\mb-ar
mada\venv\lib\site-packages\PySide2\Qt5Bluetooth.dll
11409 WARNING: lib not found: api-ms-win-core-winrt-l1-1-0.dll dependency of d:\gitprojects\mb-armada\ve
nv\lib\site-packages\PySide2\Qt5Bluetooth.dll
12009 INFO: Looking for eggs
12010 INFO: Using Python library d:\gitprojects\mb-armada\venv\scripts\python37.dll
12010 INFO: Found binding redirects:
[]
12014 INFO: Warnings written to D:\GitProjects\mb-armada\build\armada\warn-armada.txt
12062 INFO: Graph cross-reference written to D:\GitProjects\mb-armada\build\armada\xref-armada.html
12142 INFO: checking PYZ
12143 INFO: Building PYZ because PYZ-00.toc is non existent
12143 INFO: Building PYZ (ZlibArchive) D:\GitProjects\mb-armada\build\armada\PYZ-00.pyz
12879 INFO: Building PYZ (ZlibArchive) D:\GitProjects\mb-armada\build\armada\PYZ-00.pyz completed succes
sfully.
12891 INFO: checking PKG
12891 INFO: Building PKG because PKG-00.toc is non existent
12891 INFO: Building PKG (CArchive) PKG-00.pkg
12906 INFO: Building PKG (CArchive) PKG-00.pkg completed successfully.
12907 INFO: Bootloader d:\gitprojects\mb-armada\venv\lib\site-packages\PyInstaller\bootloader\Windows-64
bit\run.exe
12908 INFO: checking EXE
12908 INFO: Building EXE because EXE-00.toc is non existent
12908 INFO: Building EXE from EXE-00.toc
12909 INFO: Appending archive to EXE D:\GitProjects\mb-armada\build\armada\armada.exe
12912 INFO: Building EXE from EXE-00.toc completed successfully.
12916 INFO: checking COLLECT
12917 INFO: Building COLLECT because COLLECT-00.toc is non existent
12917 INFO: Building COLLECT COLLECT-00.toc
16576 INFO: Building COLLECT COLLECT-00.toc completed successfully.

```
